client/src/components/PanelContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+
+import Panel from "./Panel";
+
+function PanelContainer({ content = [] }) {
+  return !content.length ? (
+    <NoContent>Repo man is not here</NoContent>
+  ) : (
+    <Container>
+      {content.map((panelProps, index) => (
+        <Panel key={`${panelProps.name}${index}`} {...panelProps} />
+      ))}
+    </Container>
+  );
+}
+
+const Container = styled.div`
```

Comment:
I think it's just luck and how the code gets transpiled, in declaring Components after usage.